src/deepsparse/server/main.py

```diff
@@ -152,7 +152,7 @@ def _create_endpoint(endpoint_path: str, from_files: bool = False):
         async def _predict_func(request: input_schema):
             if from_files:
                 request = pipeline.input_schema.from_files(
-                    file.file for file in request
+                    files=[file.file for file in request], remove_masks_from_output=True
```

Comment:
this kwarg is specific to the yolact schema, we shouldn't be assigning it in the generic server main (ie it will break this line https://github.com/neuralmagic/deepsparse/blob/6c66c27860521a062c6edad7430178dd8bd86fd9/src/deepsparse/pipelines/computer_vision.py#L67).

instead - can we add a named kwarg to `from_files`, such as `from_server: bool` and let the yolact `from_files` overwrite `return_masks` directly?